### DISCUSSION TITLE
Duda T1 Complejidad mediana

### DISCUSSION THREAD

**Discussion by ncruzm** [IIC2133-PUC/2024-2#87] [Tarea 1] [answered]
Hola! Llevo varios días logrando buscar una forma de cumplir la restricción de tiempo O(log(n)) en particular me complica la parte de la mediana. Si bien logre encontrar una forma de obtener la mediana en tiempo logaritmico, luego lo que conlleva este metodo igual me fuerza a un mayor tiempo. Sera posible que me puedan orientar un poco o darme alguna fuente donde poder indagar para poder lograrlo? que me tiene muy frenado.

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, el problema de encontrar medianas a través de heaps no es algo nuevo, existe diversa literatura que puedes encontrar en internet sobre eso, lamentablemente no podemos dar más ayuda que eso, saludos.

**Reply by Utmite** [reply]
Hola Buenas,
Encontré un texto que utiliza heaps para encontrar la mediana y el minimo. Al hacer el informe, ¿debo citar dicho texto?
Muchas gracias <3

**Reply by tomastrivino** [reply]
Hola, @Utmite asegúrate de incluir la bibliografía necesaria (al menos como referencia).
